client/src/document.js

```diff
@@ -133,47 +133,74 @@ function SidebarLeaf({ depth, title, content }) {
   );
 }
 
-function RenderHTMLElementDocument({ document }) {
-  let sections = [];
-
-  sections.push(
-    <Prose key="short_description" section={document.prose.short_description} />
-  );
-  sections.push(
-    <InteractiveExample key="interactive_example" document={document} />
-  );
-  sections.push(<Prose key="overview" section={document.prose.overview} />);
-  sections.push(<Attributes key="attributes" document={document} />);
-  sections.push(
-    <ProseWithHeading key="usage_notes" section={document.prose.usage_notes} />
-  );
-  sections.push(
-    <ProseWithHeading
-      key="accessibility_concerns"
-      section={document.prose.accessibility_concerns}
-    />
-  );
-  sections.push(<Examples key="examples" document={document} />);
-  sections.push(
-    <BrowserCompatibility key="browser_compatibility" document={document} />
-  );
-  sections.push(
-    <ProseWithHeading key="see_also" section={document.prose.see_also} />
-  );
+function RenderDocumentBody({ document }) {
+  const sections = [];
+  /**
+   * The reason we can't return a filtered map of 'document.body' is because
+   * some of the parts of 'document.body' is an array.
+   * E.g. document.body.additional_prose == [
```

Comment:
Hmm, I guess not technically a bug but this is something we should revisit on the content side. I can't think of a reason that it shouldn't be possible to consume additional prose sections in the exact same way you'd consume regular prose sections.